airflow/providers/apache/iceberg/hooks/iceberg.py

```diff
@@ -23,30 +23,30 @@
 
 from airflow.hooks.base import BaseHook
 
-DEFAULT_TABULAR_URL = "https://api.tabulardata.io/ws/v1"
+DEFAULT_ICEBERG_URL = "https://api.tabulardata.io/ws/v1"
```

Comment:
Probably we want to get rid of this default :)

Reply:
Does the iceberg have a default URL?

Reply:
No, it needs to point to your catalog, so that differs per organization